build/docker/crossbuild.dockerfile

```diff
@@ -0,0 +1,14 @@
+# As a basic image for building various components of KubeEdge
+FROM ubuntu:18.04
+RUN apt-get update && apt-get install -y wget \
+    vim git make gcc \
+    upx-ucl gcc-aarch64-linux-gnu libc6-dev-arm64-cross gcc-arm-linux-gnueabi libc6-dev-armel-cross &&\
+    apt-get autoremove -y &&\
+    apt-get clean &&\
+    rm -rf /var/lib/apt/lists/*
+RUN wget -c https://dl.google.com/go/go1.17.linux-amd64.tar.gz -O - | tar -xz -C /usr/local
```

Comment:
```
As a basic image for building various components of KubeEdge.
used for edgecore and keadm multi arch build.
```
I'm not quite sure about how this Dockerfile worked. 
But here we emphasize the golang arch: linux-amd64, Is it can be used for image build and cross platform image build?

Reply:
This is the basic image used to build the `edgecore` multi-architecture binary, not the basic image used to build the multi-architecture image.
Deploying edgecore adopts a process deployment method, which requires binary rather than running the image directly.